src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.Unix.cs

```diff
@@ -51,7 +51,8 @@ private static void LoadAppLocalIcuCore(ReadOnlySpan<char> version, ReadOnlySpan
             IntPtr icuucLib = LoadLibrary(CreateLibraryName("libicuuc", suffix, extension, version, versionAtEnd), failOnLoadFailure: true);
             IntPtr icuinLib = LoadLibrary(CreateLibraryName("libicui18n", suffix, extension, version, versionAtEnd), failOnLoadFailure: true);
 
-            Interop.Globalization.InitICUFunctions(icuucLib, icuinLib, version, suffix);
+            // if suffix.length == 1 it means there is no suffix but we added the '.' separator.
```

Comment:
@tarekgh I found a small bug on App Local ICU in Unix when no suffix is passed down, since we convert it to a '.', it tries to use that as the suffix and therefore it doesn't find any exported symbols.

Reply:
I suggest to have it in its own PR and link this PR to previous ones. This will good to see only the changes related to the local ICU. I am ok to keep it here too if you feel strongly about it.

---
In reply to: [435546383](https://github.com/dotnet/runtime/pull/37281#discussion_r435546383) [](ancestors = 435546383)

Reply:
I can move it to a separate PR. The reason why I needed it here was to test against a newer and an older ICU, so I used the App-Local feature.

Will move it to it's own PR 😄

Reply:
as I mentioned if you want to keep it, that is fine. I won't add more work on you :-)

---
In reply to: [435549763](https://github.com/dotnet/runtime/pull/37281#discussion_r435549763) [](ancestors = 435549763)

Reply:
>  I won't add more work on you :-)

The fix needs a little bit more work, so I'll put it in a separate PR.

Reply:
I opened: https://github.com/dotnet/runtime/pull/37455